ang/civicase/case/actions/services/email-managers-case-action.service.js

```diff
@@ -3,12 +3,17 @@
 
   module.service('EmailManagersCaseAction', EmailManagersCaseAction);
 
+  /**
+   * EmailManagersCaseAction service callback function.
+   */
   function EmailManagersCaseAction () {
     /**
-     * Click event handler for the Action
+     * Returns the configuration options to open up a mail popup to
+     * communicate with the case managers. Displays an error message
+     * when no case managers have been assigned to the case.
      *
      * @param {Array} cases
-     * @param {Object} action
+     * @param {object} action
```

Comment:
are you not getting error while committing these changes? Because the param description is missing.

Reply:
No. These changes are being created using husky linters config only. Let's have a call whenever you have time